OracleDatabase/SingleInstance/README.md

```diff
@@ -180,6 +184,49 @@ Archive mode can be enabled during the first time when database is created by se
 
 In case this parameter is set `true` and passed to `docker run` command while reusing existing datafiles, even though this parameter would be visible as set to `true` in the container environment, this would not be set inside the database. The value used at the time of database creation will be used.
 
+#### Configuring TCPS in the database (Support from version 19.3.0 onwards)
+There are two ways to enable TCPS in the database:
+1. Enable TCPS while creating the database.
+2. Enable TCPS after the database is created.
+
+To enable TCPS while creating the database, use `-e ENABLE_TCPS=true` option with the `docker run` command. A listener endpoint will be created at the container port 1522 for TCPS.
+
+To enable TCPS after the database is created, please use one the following sample command:
+```bash
+    # Creates Listener for TCPS at container port 1522
+    docker exec -it <container name> /opt/oracle/configTcps.sh
+```
+
+Similarly, to disable TCPS in the database, please use the following command:
+```bash
+    # Disable TCPS in the database
+    docker exec -it <container name> /opt/oracle/configTcps.sh disable
+```
+
+**NOTE**:
+- Only the database server authentication is supported (no mTLS).
+- The container port at which TCPS listener is listening (i.e. 1522) should be exposed and mapped to some host port using `-p <host-port>:1522` option in the `docker run` command. It is required to connect to the database from the outside world using TCPS.
+- When TCPS is enabled, a self-signed certificate will be created for the same. For user's convenience, a client side wallet is prepared and stored at the location, `/opt/oracle/oradata/clientWallet/$ORACLE_SID`. You can use this client wallet along with SQL\*Plus to connect with the database. The sample command to download the client wallet is as follows:
+    ```bash
+        # ORACLE_SID default value is ORCLCDB
+        docker cp <container name>:/opt/oracle/oradata/clientWallet/<ORACLE_SID> <destination directory>
+    ```
+- The client wallet above will have wallet files, along with sample `sqlnet.ora` and `tnsnames.ora`. You should edit the `HOST` and `PORT` field accordingly in the `tnsnames.ora` before connecting using TCPS.
+- After the `tnsnames.ora` is modified, go inside the downloaded client wallet directory and set the TNS_ADMIN for SQL\*Plus by using the `export TNS_ADMIN=$(pwd)` command. Then user can connect with the following sample commands:
```

Comment:
Change 'After the' to 'After' and 'the TNS_ADMIN' to 'TNS_ADMIN'.

Reply:
Fixed.